### PR TITLE
Adds HemisphereLight descriptor

### PR DESCRIPTION
**Pull request by kevzettler**
This PR adds a descriptor for HemisphereLight

### CODE REVIEW

File: src/lib/descriptors/Light/LightDescriptorBase.js [hunk 1]
```diff
@@ -188,10 +188,10 @@ class LightDescriptorBase extends Object3DDescriptor {
     }
   }
 
-  hasColor() {
+  hasColor(colorType = 'color') {
```
Comment (kevzettler) [inline src/lib/descriptors/Light/LightDescriptorBase.js]: will this work with the `es` version being used?

**Comment by kevzettler** [comment]
How do I add tests for this?

**Comment by kevzettler** [comment]
ping @toxicFork

File: src/lib/ElementDescriptorContainer.js [hunk 2]
```diff
@@ -61,7 +62,6 @@ import BezierCurveToDescriptor from './descriptors/Geometry/Shapes/BezierCurveTo
 import QuadraticCurveToDescriptor from './descriptors/Geometry/Shapes/QuadraticCurveToDescriptor';
 import AbsArcDescriptor from './descriptors/Geometry/Shapes/AbsArcDescriptor';
 import AbsEllipseDescriptor from './descriptors/Geometry/Shapes/AbsEllipseDescriptor';
-import HoleDescriptor from './descriptors/Geometry/Shapes/HoleDescriptor';
```
Comment (toxicFork) [inline src/lib/ElementDescriptorContainer.js]: I think it may have accidentally been removed, see below

Comment (kevzettler) [inline src/lib/ElementDescriptorContainer.js]: @toxicFork I intentionally removed it below. It was commented out and triggering Eslint. If it is intended to be included I can uncomment it here.

Comment (kevzettler) [inline src/lib/ElementDescriptorContainer.js]: ok you're correct looks like it wasn't commented as I thought I might have accidently done that localy. I will restore

File: src/lib/descriptors/Light/LightDescriptorBase.js (hunk 1 above)
Comment (toxicFork) [inline src/lib/descriptors/Light/LightDescriptorBase.js]: late again (i shouldn't promise days for delivery) but it's fine, I use it for material descriptor too :)

**Comment by toxicFork** [comment]
Merging into staging/3.0.

Will use "skyColor" property instead of "color"

**Pull request merged**